spacy/tokens/doc.pyx

```diff
@@ -1462,6 +1462,135 @@ cdef class Doc:
                 remove_label_if_necessary(attributes[i])
                 retokenizer.merge(span, attributes[i])
 
+    def from_json(self, doc_json):
+        """Convert a JSON document generated by Doc.to_json() to a Doc.
+
+        vocab (Vocab): A storage container for lexical types.
+        RETURNS (Doc): A doc instance corresponding to the specified JSON representation.
+        """
```

Comment:
I think we can add an option `, *, validate=False` option to use the schema for additional validation and then this is stable enough that you can go head and also update the docstring.